src/hooks/useVenueId.ts

```diff
@@ -11,7 +11,11 @@ const venuePaths = [
   "/v/:venueId",
   "/admin_v2/venue/:venueId",
 ];
-export const useVenueId: () => string | null = () => {
+
+// @debt I think we could just use useParams() here instead of needing to loop through all of the venuePath, so long as the use :venueId in their path
+//   https://reactrouter.com/web/api/Hooks/useparams
+//     useParams returns an object of key/value pairs of URL parameters. Use it to access match.params of the current <Route>.
+export const useVenueId: () => string | undefined = () => {
```

Comment:
[OOS] I think we could just use `useParams()` here instead of needing to loop through all of the `venuePath`s, so long as they use `:venueId` in their path

- https://reactrouter.com/web/api/Hooks/useparams
  - > `useParams` returns an object of key/value pairs of URL parameters. Use it to access `match.params` of the current `<Route>`

Reply:
backlogged in https://github.com/sparkletown/internal-sparkle-issues/issues/164